src/components/Icon/Illustrations.ts

```diff
@@ -102,7 +101,6 @@ export {
     LaptopWithSecondScreenSync,
     LaptopWithSecondScreenX,
     MoneyIntoWallet,
-    TreasureChest,
```

Comment:
I don't know if we have to remove these from this file. Can you please confirm on the issue?

Reply:
It's okay if all `TreasureChest` usage is migrated.